core/test.ts

```diff
@@ -24,7 +25,26 @@ export class Test {
     return this;
   }
 
-  public dataset(datasetToTest: string) {
+  public dataset(reference: string | IResolvable) {
+    var datasetToTest = null;
```

Comment:
i don't think we need this code - let's just assign `reference` to `this.datasetToTest` and resolve the dataset inside `compile()`

Reply:
Done